python/semantic_kernel/kernel.py

```diff
@@ -47,9 +47,14 @@
 from semantic_kernel.services.ai_service_selector import AIServiceSelector
 from semantic_kernel.utils.validation import validate_plugin_name
 
+if TYPE_CHECKING:
+    from semantic_kernel.connectors.ai.chat_completion_client_base import ChatCompletionClientBase
+    from semantic_kernel.connectors.ai.embeddings.embedding_generator_base import EmbeddingGeneratorBase
+    from semantic_kernel.connectors.ai.text_completion_client_base import TextCompletionClientBase
+
 T = TypeVar("T")
 
-ALL_SERVICE_TYPES = "TextCompletionClientBase | ChatCompletionClientBase | EmbeddingGeneratorBase"
+ALL_SERVICE_TYPES = Union["TextCompletionClientBase", "ChatCompletionClientBase", "EmbeddingGeneratorBase"]
```

Comment:
This can't get away with a `|` instead of a Union?

Reply:
nope...